### PR TITLE
release notes 2021.11

### LINKED ISSUE #6221: release notes 2021.11
**Describe the issue. What is the expected and unexpected behavior?**
release notes 2021.11

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #6221 

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,90 @@ id: Release notes
...
+- **Avatar:**
+  - Updated example image to match core ([#6137](https://github.com/patternfly/patternfly-react/pull/6137))
+- **Breadcrumb**
+  - Fixed ts-lint errors ([#6113](https://github.com/patternfly/patternfly-react/pull/6113))
+- **Card:**
+  - Added horizontal split and details demos ([#6193](https://github.com/patternfly/patternfly-react/pull/6193))
+- **Context selector:**
+  - Added plain variant ([#6141](https://github.com/patternfly/patternfly-react/pull/6141))
+- **Dual list selector:**
+  - Fixed to respect initial chosen options, allow checking of sub folders, and reset selected states when options are moved via add/remove al([#6180](https://github.com/patternfly/patternfly-react/pull/6180))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:30]: Change to `add/remove all (`

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,90 @@ id: Release notes
...
+- [@patternfly/react-table@4.29.58](https://www.npmjs.com/package/@patternfly/react-table/v/4.29.58)
+- [@patternfly/react-tokens@4.12.15](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.12.15)
+- [@patternfly/react-topology@4.9.62](https://www.npmjs.com/package/@patternfly/react-topology/v/4.9.62)
+- [@patternfly/react-virtualized-extension@4.9.31](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.9.31)
+
+### Components
+- **Avatar:**
+  - Updated example image to match core ([#6137](https://github.com/patternfly/patternfly-react/pull/6137))
+- **Breadcrumb**
+  - Fixed ts-lint errors ([#6113](https://github.com/patternfly/patternfly-react/pull/6113))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:24]: Maybe move this to chore?

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,90 @@ id: Release notes
...
+- **Menu:**
+  - Added application launcher demo ([#6175](https://github.com/patternfly/patternfly-react/pull/6175))
+  - Fixed menu crash if current menuRef value is not found. ([#6152](https://github.com/patternfly/patternfly-react/pull/6152))
+- **Navigation:**
+  - Added horizontal sub navigation variant ([#6149](https://github.com/patternfly/patternfly-react/pull/6149))
+- **Number input:**
+  - Fixed to only call 'onPlus' and 'onMinus' if set ([#6197](https://github.com/patternfly/patternfly-react/pull/6197))
+- **Popover:**
+  - Updated to call setVisible before animation ([#6104](https://github.com/patternfly/patternfly-react/pull/6104))
+  - Addedd support for diagonal positions ([#6155](https://github.com/patternfly/patternfly-react/pull/6155))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:46]: Extra d in "Addedd"

```suggestion
  - Added support for diagonal positions ([#6155](https://github.com/patternfly/patternfly-react/pull/6155))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,90 @@ id: Release notes
...
+- **Search input:**
+  - Added support for submit button ([#6178](https://github.com/patternfly/patternfly-react/pull/6178))
+  - Updated to submit the search when hitting enter in the main input field ([#6145](https://github.com/patternfly/patternfly-react/pull/6145))
+- **Select:**
+  - Fixed typeahead children check ([#6176](https://github.com/patternfly/patternfly-react/pull/6176))
+  - Omit isLastOptionBeforeFooter from props ([#6147](https://github.com/patternfly/patternfly-react/pull/6147))
+- **Time picker:**
+  - Fixed so 0 & 00 are no longer a valid hour values in 12 hour time. Updated onChange handler to return flag indicating if current time value is valid. ([#6134](https://github.com/patternfly/patternfly-react/pull/6134))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:54]: Small nit
```suggestion
  - Fixed 0 & 00 are no longer valid hour values in 12 hour time. Updated onChange handler to return flag indicating if current time value is valid. ([#6134](https://github.com/patternfly/patternfly-react/pull/6134))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
2 small nits - thanks!

**Comment by nicolethoen** [comment]
Can you also update snapshots?

**Review by redallen — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**